repldex/discordbot/bot.py

```diff
@@ -142,20 +138,21 @@ async def process_commands(self, message):
 					continue
 			else:
 				return_args = []
-			for _attempt in range(10):
+			for _ in range(10):
 				try:
 					return await func(ctx, *return_args)
 				except TypeError:
 					if pad_none:
 						return_args.append(None)
 					else:
 						break
-				except BaseException:
+				except Exception:
```

Comment:
This catches BaseException instead of Exception because NothingFound extends BaseExtension, the correct change would be to change `class NothingFound(BaseException)` to `class NothingFound(Exception)`